doc/man3/EVP_EncryptInit.pod

```diff
@@ -667,6 +667,15 @@ EVP_EncryptFinal_ex(), EVP_DecryptInit_ex(), EVP_DecryptFinal_ex(),
 EVP_CipherInit_ex() and EVP_CipherFinal_ex() because they can reuse an
 existing context without allocating and freeing it up on each call.
 
+There are some differences between functions EVP_CipherInit() and
+EVP_CipherInit_ex(), significant in some circumstances. EVP_CipherInit() fills
+the passed context object with zeros. It means that the flags set for the CTX
+are removed, and it is especially important for the
+B<EVP_CIPHER_CTX_FLAG_WRAP_ALLOW> flag treated specially in
+EVP_CipherInit_ex(). As a consequence, EVP_CipherInit() does not allow
```

Comment:
These two are a little bit unrelated. Or rather I would switch the order of these two sentences: ...fills the passed context object with zeros. As a consequence, EVP_CipherInit() does not allow....calls. It also means that flags set for the CTX prior to the call are removed and that is especially important...

Reply:
Fixed. Thanks!